documentation/how-to/!how-to-template.md

```diff
@@ -0,0 +1,139 @@
+---
+title: {Guide Title for a Specific Task}
+---
+<!-- template preamble -->
+
+## Template preamble (remove!)
+
+A how-to page shows how to do a single thing, typically by giving a short
+sequence of steps. How-to pages have minimal explanation, but often provide links
+to conceptual topics that provide related background and knowledge.
+
+How-tos should provide the available environments (`test`/`prod`) and endpoints, when applicable.
+
+To write a new how-to page, copy this file. All text in {curly brackets} should be replaced or removed.
+
+For more information on how-to pages, read the [Good Docs Project guide](https://gitlab.com/tgdp/templates/-/blob/main/how-to/guide-how-to.md).
+
+---
+
+<!-- overview -->
+
+This guide explains how to {insert a brief description of the task}.
+
+{Optional: Specify when and why your user might want to perform the task.}
+
+{Optional: Include links/tooltips to relevant concepts when terms are introduced here or in the `steps` section.}
+
+<!-- prerequisites -->
+
+## Before you begin
+
+{Optional: You should be familiar with how to do a [more basic task](/how-to/gwa-install.md) or [concept](/concepts/api-directory.md).}
+
+Before you begin, ensure you:
+
+- [Install gwa CLI](/how-to/gwa-install.md)
+- [Create a Namespace](/resources/gwa-commands.md#namespacecreate)
+- [Complete This](/how-to/create-gateway-service.md)
+- [Do That](/how-to/generate-service-account.md)
+
+<!-- steps -->
+
+## {Sub-task name}
+
+{Optional: Provide a concise description of the purpose of this sub-task. Only include this if the purpose is not clear from the sub-task title.}
+
+{You can use this format to describe steps to complete a sub-task:}
+
+1. {Write the first step here. Use a verb to start.}
+
+  {Optional: Explanatory text}
+
+  {Optional: Code sample or screenshot that helps your users complete this step.}
+
+  {Optional: The result of completing this step.}
+
+1. {Write the second step here. Use a verb to start.}
+
+  {Substep 1 text}
+
+  {Substep 2 text}
+
+## {Second sub-task name}
+
+{Repeat as above}
+
+{Note on multiple paths: Making users choose between multiple ways of doing this
+ is confusing. When it comes to CLI vs UI, the CLI is generally the recommended
```

Comment:
we should emphasize if using multiple paths it should be made **very clear that this is an alternative way of completing the same task** to limit confusion - note historically there has been confusion around using the CLI vs UI

Reply:
For sure. What do you think about the guidance to avoid multiple paths? (and thus no additional clarification is needed)